elements/pfe-jump-links/src/pfe-jump-links-nav.scss

```diff
@@ -0,0 +1,354 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+
+$LOCAL: jump;
+
+$variables: (
+  // Padding: calc(#{pfe-var(container-spacer)} * 2),
+  // BackgroundColor: pfe-color(surface--base),
+);
+
+:host {
+  // Disable accordion's active borders
+  --pfe-accordion--BackgroundColor: transparent;
```

Comment:
Can we scope these to pfe-accordion since some of the variables are global theme variables and could potentially impact slotted content otherwise?

Reply:
For sure!